apps/graphql/src/services/testingTools/TestingTools.js

```diff
@@ -2,10 +2,10 @@
 
 import { graphql as originalGraphQL } from 'graphql';
 
-import schema from '../Schema';
-import createContext from './GraphQLContext';
+import schema from '../../Schema';
+import createContext from '../graphqlContext/GraphQLContext';
 
-jest.mock('./Fetch.js');
+jest.mock('../fetch/Fetch.js');
```

Comment:
Can you try and remove this mock and see if tests still work? We already mock fetch in `setupJest.js` (I know it is not related to this PR, but anyways 👼 )

Reply:
Sure, I'll try that!

Reply:
Good catch, it works well without this line.